src/Microsoft.Identity.Client/Features/ConfidentialClient/ConfidentialClientApplication.cs

```diff
@@ -389,6 +389,19 @@ async Task<AuthenticationResult> IConfidentialClientApplicationWithCertificate.A
             return await handler.CreateAuthorizationUriAsync().ConfigureAwait(false);
         }
 
+        /// <summary>
+        /// This method should be used when you have a solution using ADAL 2.x and 
+        /// caching a refresh token, and you want to migrate to MSAL.NET.
+        /// During the migration process, it enables you to store in MSAL.NET token
+        ///  cache an access token and refresh token corresponding to <paramref="refreshToken"/>.
```

Comment:
As you called out, the access token is not returned.

Reply:
Sadly C# does not support inheritdoc like Java does. We can use a See <cref="IFoo.Bar" /> to send people to the docs on the interface though. 

https://stackoverflow.com/questions/342964/comment-inheritance-for-c-sharp-actually-any-language